gnocchi/cli/metricd.py

```diff
@@ -162,7 +162,8 @@ def _configure(self):
         try:
             self.partitioner = self.coord.join_partitioned_group(
                 self.GROUP_ID, partitions=200)
-            LOG.info('Joined coordination group: %s', self.GROUP_ID)
+            LOG.info('Joined coordination group: %s',
+                     self.GROUP_ID.decode('utf8'))
```

Comment:
Can you just use `decode()` without arg? It ought to work I think and allows to use whatever encoding is available rather than forcing an encoding we don't know about.

Reply:
Yes - that works fine - updated.